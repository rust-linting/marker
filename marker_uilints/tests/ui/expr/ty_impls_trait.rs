#![warn(marker::marker_uilints::test_ty_impls_trait)]

fn main() {
    let check_traits = 1;
    let _ = check_traits;
}