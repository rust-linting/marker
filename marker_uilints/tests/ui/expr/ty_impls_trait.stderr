warning: checking trait impls:
 --> $DIR/ty_impls_trait.rs:5:13
  |
5 |     let _ = check_traits;
  |             ^^^^^^^^^^^^
  |
  = note: Implements: `std::clone::Clone`: false
note: the lint level is defined here
 --> $DIR/ty_impls_trait.rs:1:9
  |
1 | #![warn(marker::marker_uilints::test_ty_impls_trait)]
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: 1 warning emitted

